libc/cmake/modules/LLVMLibCTestRules.cmake

```diff
@@ -533,15 +542,14 @@ endfunction(add_integration_test)
 set(LIBC_HERMETIC_TEST_COMPILE_OPTIONS ${LIBC_COMPILE_OPTIONS_DEFAULT}
```

Comment:
this section was moved into `LLVMLibcCCompileOptionRules.cmake` in https://github.com/llvm/llvm-project/pull/81917

Reply:
Yeah there's a lot of logic that's moved and broken after rebasing. Trying to figure out what's changed.

Reply:
it should be just the part here and the part relating to compile options in libc object rules